### ISSUE TITLE
Pressing any number key doesn't exit pasting transient state

### ISSUE THREAD

**Issue by Andre0991** [syl20bnr/spacemacs#4930]
#### Description
Pressing any number key doesn't exit pasting transient state.

#### Reproduction guide
- Start Emacs
- Paste something with `p`
- Press `0`

*Observed behaviour:*
Nothing happens.

*Expected behaviour:*
Point should move to the beginning of line.

#### System Info
- OS: darwin
- Emacs: 24.5.1
- Spacemacs: 0.105.9
- Spacemacs branch: develop (rev. 18874c9)
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(osx lispy spacemacs-helm emacs-lisp syntax-checking common-lisp git github
     (shell :variables shell-default-shell 'eshell)
     spacemacs-layouts erc html pdf-tools
     (elfeed :variables rmh-elfeed-org-files
             (list "/Users/andre0991/Dropbox/backup/emacs/elfeed-feeds.org"))
     themes-megapack semantic
     (auto-completion :variables auto-completion-private-snippets-directory "/Users/andre0991/Dropbox/backup/emacs/yasnippet-snippets/" auto-completion-return-key-behavior nil auto-completion-tab-key-behavior 'complete auto-completion-enable-help-tooltip t auto-completion-enable-sort-by-usage t auto-completion-enable-snippets-in-popup t :disabled-for latex)
     org
     (ranger :variables ranger-show-preview t)
     markdown latex spell-checking version-control andre-common-lisp andre-elfeed andre-osx andre-erc andre-eww)
```

Thanks.

**Comment by AlejandroCatalina** [comment]
I can reproduce it. I'm looking into the transient state, but everything seems well defined, there is no action bound to any number key, which should make it exit the state. I'm afraid this is beyond my reach, as long as the macro `spacemacs|define-transient-state` also seems okay to me. This might have something to do with the keymap defined inside the transient state?

**Comment by Andre0991** [comment]
Thanks @AlejandroCatalina :+1: 

@StreakyCobra Can you tag this issue as PR pending please?

**Comment by StreakyCobra** [comment]
@Andre0991 Done thx :+1:

**Comment by d12frosted** [comment]
Fixed with released of Spacemacs v0.200. Let us know if you still have any problems with this issue.

**Comment by d12frosted** [comment]
Fixed with released of Spacemacs v0.200. Let us know if you still have any problems with this issue.
